CHANGELOG.md

```diff
@@ -10,10 +10,12 @@
 
 ### Fixed
 
-- `Input`: controlled Inputs could get ouf sync when the controlled value was changed outside of the component (thus not via the onChange handler) ([@lowiebenoot](https://github.com/lowiebenoot)) in [#2271](https://github.com/teamleadercrm/ui/pull/2271))
-
 ### Dependency updates
 
+## [15.0.2] - 2022-07-11
+
+- `Input`: controlled Inputs could get ouf sync when the controlled value was changed outside of the component (thus not via the onChange handler) ([@lowiebenoot](https://github.com/lowiebenoot)) in [#2271](https://github.com/teamleadercrm/ui/pull/2271))
```

Comment:
I think it should be under `### Fixed` again.

Reply:
Ah damn, indeed 🙈